test/unit/controllers/AuthSpec.scala

```diff
@@ -47,22 +47,15 @@ class AuthSpec extends ControllerSpec with BeforeAndAfterEach {
       val result = controller.show(request)
       status(result) shouldBe SEE_OTHER
       redirectLocation(result) should contain("/manage-email-cds/change-email-address/create")
-
-    }
-
-    "not allow an authorised user without an enrolled eori to access the page" in withAuthorisedUserWithoutEori {
-      val result = controller.show(request)
-      status(result) shouldBe SEE_OTHER
-      redirectLocation(result) should contain(ineligibleUrl)
     }
 
-    "not allow an authorised user with an empty eori to access the page" in withAuthorisedUserWithoutEori {
+    "not allow an authorised user with and eori and without an internal id to access the page" in withUnauthorisedUserWithoutInternalId {
```

Comment:
unnecessary and work before eori, actualy how about -> "...with eori but without an internal id

Reply:
done @Jodlowy